viewer/templates/viewer/help.html

```diff
@@ -138,13 +146,47 @@ <h2>HTML</h2>
         Search "&lt;br&gt;"
       </a>
     </li>
-
     <li class="m-list_item">
       You want to find pages that include a specific image filename.
       <a href="{% url 'index' %}?search_type=html&q=consumer_tools_hero">
         Search "consumer_tools_hero"
       </a>
     </li>
+    <li class="m-list_item">
+      You want to find pages encoded in Traditional Chinese.
+      <a
+        href="{% url 'index' %}?search_type=html&q={{ 'html lang=&quot;zh-Hant&quot;' | urlencode }}">
```

Comment:
Unfortunately this change breaks what is being searched. This seems to work as an alternative:

```suggestion
      <a href="{% url 'index' %}?search_type=html&q=html+lang=%22zh-Hant%22">
```